pyproject.toml

```diff
@@ -0,0 +1,68 @@
+[build-system]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "tomato-alicat"
+version = "0.1.0"
+description = "Package dedicated to alicat devices for tomato."
+readme = "README.md"
+requires-python = ">=3.8"
+
+authors = [
+    {Kraus Peter , peter.kraus@ceramics.tu-berlin.de,
+    Gbocho Alexandre , gbocho@tu-berlin.de }]
+
+keywords = ["automation", "package", "alicat", "tomato"]
+license = { GNU General Public License v3.0  }
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: MIT License"
+]
+
+dependencies = [
+    "aioconsole==0.8.1",
+    "alicat==0.6.2",
```

Comment:
`alicat` should be the only dependency. Please do not pin it like this, `alicat >= 0.6.2` is better.